src/rns/index.ts

```diff
@@ -61,7 +61,7 @@ async function precache (eth?: Eth): Promise<void> {
   await fetchEventsForService(eth, 'rns.placement', simplePlacementsContractAbi as AbiItem[], dataQueuePusher)
 
   // We need to sort the events in order to have valid
-  eventsDataQueue.sort((a, b): number => {
+  const sortedQueue = eventsDataQueue.sort((a, b): number => {
```

Comment:
This assignment is unnecessary, they both point to the same array as `Array.sort` is in place and modifies original array https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
Right, I thought it is in place, but when I was checking it I just saw the "Return value = sorted array" and did not finish reading to verify that it was in place 🙈 Thanks for the catch